src/Pos/Context/Context.hs

```diff
@@ -108,6 +109,8 @@ data NodeContext ssc = NodeContext
     -- ^ Context needed for the update system
     , ncLrcContext          :: !LrcContext
     -- ^ Context needed for LRC
+    , ncDiscoveryContext    :: !DiscoveryContextSum
```

Comment:
Do I understand correctly that:
1. It's not used anywhere at all
2. It can be retrieved with `Ether.ask` anyway